ui/src/jobs/details/config/source_config_section/bq_config_section/BigQueryDatasetSection.scss

```diff
@@ -6,5 +6,4 @@
 
 .euiBadgeGroup---insideDescriptionList {
   padding: 4px 0;
-  margin-left: -8px;
```

Comment:
This is no longer required because we're setting the column gap to 0px here https://github.com/caraml-dev/turing/pull/384/files#diff-d59e126a6e6751d23dc2750464fc39dcfef504cd0068bbca584f36705ad42b4bR13.

This change ensures that the collapsible badge group are flushed to the left edge:
![Screenshot 2024-07-15 at 17 36 49](https://github.com/user-attachments/assets/fbd3189a-6f25-47a0-aa0b-e19825b9196d)